src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <div class="flex-horizontal flex-wrap margin-right">
                 <div>
-                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter">
+                    <input class="form-control" placeholder="Filter" data-bind="textInput: filter" title="Filter 'Custom Info' & 'URI' columns">
```

Comment:
can we put that info to placeholder instead to title? If it fits

Reply:
![image](https://user-images.githubusercontent.com/17376974/70434496-0718d400-1a8d-11ea-87fd-4768e3d50297.png)

Reply:
please make input wider?

Reply:
done